crates/ibc/src/core/ics24_host/identifier/validate.rs

```diff
@@ -219,13 +188,6 @@ mod tests {
         assert!(id.is_err())
     }
 
-    #[test]
-    fn parse_invalid_id_empty() {
-        // invalid id empty
-        let id = validate_identifier_chars("");
-        assert!(id.is_err())
-    }
-
```

Comment:
Instead of deleting this test, I would modify this test with `validate_identifier_length`.

Actually, for any breaking changes, you should add tests that check the change. So for this PR, you should test
- `validate_identifier_chars("")` is successful.
- `validate_identifier_length("", 0, 0)` returns an error.
- `validate_{identifier,prefix}_length` don't panic anymore if `min > max`